terraform/testcases/ecshealthcheck/ecs_taskdef.tpl

```diff
@@ -0,0 +1,148 @@
+[
+    {
+      "name": "${sample_app_container_name}",
+      "image": "${data_emitter_image}",
+      "cpu": 10,
+      "memory": 256,
+      "portMappings": [
+          {
+            "containerPort": 4567,
+            "hostPort": 4567,
+            "protocol": "tcp"
+          }
+      ],
+      "command": [],
+      "environment": [
+        {
+          "name": "OTEL_EXPORTER_OTLP_ENDPOINT",
+          "value": "http://127.0.0.1:${grpc_port}"
+        },
+        {
+           "name": "AWS_XRAY_DAEMON_ADDRESS",
+           "value": "127.0.0.1:${udp_port}"
+        },
+        {
+           "name": "COLLECTOR_UDP_ADDRESS",
+           "value": "127.0.0.1:${udp_port}"
+        },
+        {
+            "name": "AWS_REGION",
+            "value": "${region}"
+        },
+        {
+          "name": "INSTANCE_ID",
+          "value": "${testing_id}"
+        },
+        {
+        "name": "OTEL_RESOURCE_ATTRIBUTES",
+        "value": "service.namespace=${otel_service_namespace},service.name=${otel_service_name}"
+        },
+        {
+            "name": "LISTEN_ADDRESS",
+            "value": "${sample_app_listen_address}"
+        },
+        {
+        "name": "JAEGER_RECEIVER_ENDPOINT",
+        "value": "127.0.0.1:${http_port}"
+        },
+        {
+        "name": "ZIPKIN_RECEIVER_ENDPOINT",
+        "value": "127.0.0.1:${http_port}"
+        },
```

Comment:
Are these needed?  If so, they probably need to have different ports.

Reply:
Thanks. I have updated code accordingly.